<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureGraphics</class>
 <widget class="QWidget" name="ConfigureGraphics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>API Settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QWidget" name="api_widget" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="horizontalSpacing">
            <number>6</number>
           </property>
           <item row="4" column="0">
            <widget class="QWidget" name="backend_widget" native="true">
             <layout class="QHBoxLayout" name="backend_layout">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="backend_label">
                <property name="text">
                 <string>Shader Backend:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="backend"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QWidget" name="device_widget" native="true">
             <layout class="QHBoxLayout" name="device_layout">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="device_label">
                <property name="text">
                 <string>Device:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="device"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QWidget" name="api_layout_2" native="true">
             <layout class="QHBoxLayout" name="api_layout">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="api_label">
                <property name="text">
                 <string>API:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="api">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <item>
                 <property name="text">
                  <string notr="true">OpenGL</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string notr="true">Vulkan</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Graphics Settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QCheckBox" name="use_disk_shader_cache">
          <property name="text">
           <string>Use disk shader cache</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="use_asynchronous_gpu_emulation">
          <property name="text">
           <string>Use asynchronous GPU emulation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="use_nvdec_emulation">
          <property name="text">
           <string>Use NVDEC emulation</string>
          </property>
         </widget>
        </item>
        <item>
          <widget class="QCheckBox" name="accelerate_astc">
            <property name="text">
              <string>Accelerate ASTC texture decoding</string>
            </property>
          </widget>
        </item>
        <item>
         <widget class="QWidget" name="fullscreen_mode_layout" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_1">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="fullscreen_mode_label">
             <property name="text">
              <string>Fullscreen Mode:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="fullscreen_mode_combobox">
             <item>
              <property name="text">
               <string>Borderless Windowed</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Exclusive Fullscreen</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="aspect_ratio_layout" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="ar_label">
             <property name="text">
              <string>Aspect Ratio:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="aspect_ratio_combobox">
             <item>
              <property name="text">
               <string>Default (16:9)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Force 4:3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Force 21:9</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Stretch to Window</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="bg_layout" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QComboBox" name="bg_combobox">
             <property name="currentText">
              <string>Use global background color</string>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <property name="maxVisibleItems">
              <number>10</number>
             </property>
             <item>
              <property name="text">
               <string>Use global background color</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Set background color:</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="bg_label">
             <property name="text">
              <string>Background Color:</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="bg_button">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
